src/ripple/overlay/impl/ZeroCopyStream.h

```diff
@@ -56,7 +56,7 @@ class ZeroCopyInputStream
     bool
     Skip (int count) override;
 
-    std::int64_t
+    google::protobuf::int64
```

Comment:
Should we have the function return `auto` instead of `google::protobuf::int64`? This would reduce future maintenance - simply changing the declaration of `count_` would be enough.